.eslintrc

```diff
@@ -58,7 +58,7 @@
           ["^react", "^@?\\w"],
           // Internal packages.
           [
-            "^(assets|components|constants|contexts|data|hooks|i18n|images|reducers|sagas|types|utils|validations|views)(/.*|$)"
+            "^(assets|components|constants|contexts|data|hooks|i18n|images|queries|reducers|sagas|types|utils|validations|views)(/.*|$)"
```

Comment:
nit: is it worth adjusting this regex so we name the folders we _don't_ want to include?

Reply:
Good question. I'm not sure, but we can look into it.